lessons/pydata/webscraping/index.ipynb

```diff
@@ -0,0 +1,6844 @@
+{
```

Comment:
<p>potřeboval/a (nebo "až budeš potřebovat"). Asi by to mělo být genderově neutrální.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>